reconcile/utils/oc.py

```diff
@@ -439,6 +440,26 @@ def get_service_account_username(user):
         name = user.split('/')[1]
         return "system:serviceaccount:{}:{}".format(namespace, name)
 
+    def recycle_orphan_pods(self, dry_run, namespace, resource):
+        pods = self.get(namespace, 'Pod')['items']
+        for p in pods:
+            owner = self.get_obj_root_owner(namespace, p)
+            if resource.kind != owner['kind']:
+                continue
+            if resource.name != owner['metadata']['name']:
+                continue
+            # we found a pod we want to recycle
```

Comment:
I would write an INFO message here to that says which one we're recycling. I'm thinking in a situation where you have 5 pods, and the third one fails to get recycled. Then at least you know that whenever you have to fix this manually, you don't have to touch certain pods

Reply:
added logging!